File: 0344-reverse-string/0344-reverse-string.py
class Solution(object):
    def reverseString(self, s):
        """
        :type s: List[str]
        :rtype: None Do not return anything, modify s in-place instead.
        """
        n=int(len(s))
        for i in range(n/2) :
            temp=s[i]
            s[i]=s[n-i-1]
            s[n-i-1]=temp

File: 0344-reverse-string/0344-reverse-string.java
class Solution {
    public void reverseString(char[] s) {
        int n=s.length,i;
        char temp;
        for(i=0;i<n/2;i++)
        {
            temp=s[i];
            s[i]=s[n-i-1];
            s[n-i-1]=temp;
        }
    }
}